tests/qm-sanity-test/check_qm_with_aib.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Create AIB manifest with the correct QM copr
+USE_QM_COPR="${PACKIT_COPR_PROJECT:-rhcontainerbot/qm}"
```

Comment:
OK that looks good
https://download.copr.fedorainfracloud.org/results/packit/containers-qm-685/epel-9-$basearch/

Reply:
Exactly